programs/rodeos/streams/logger.hpp

```diff
@@ -6,22 +6,22 @@
 namespace b1 {
 
 class logger : public stream_handler {
-   std::vector<eosio::name> routes_;
-
  public:
-   logger(std::vector<eosio::name> routes) : routes_(std::move(routes)) { ilog("logger initialized"); }
-
-   const std::vector<eosio::name>& get_routes() const override { return routes_; }
+   explicit logger(std::vector<std::string> routes)
+   : stream_handler(std::move(routes)) {
+      ilog("logger initialized");
+   }
 
-   void publish(const std::vector<char>& data, const eosio::name& routing_key) override {
-      ilog("logger stream [${data_size}] >> ${data}", ("data", data)("data_size", data.size()));
+   void publish(const std::vector<char>& data, const std::string& routing_key) override {
+      ilog("logger stream ${r}: [${data_size}] >> ${data}",
+           ("r", routing_key)("data", data)("data_size", data.size()));
```

Comment:
The order of "data" and "data_size" is reversed.

Reply:
The order doesn't matter.